README.md

```diff
@@ -41,6 +41,152 @@ parmcards4;
 
 You now have a simple web app with a backend service!
 
+## Detailed Overview
+
+The SASjs adapter is a JS library and a set of SAS Macros that handle the communication between the frontend app and backend SAS services.
+
+There are three parts to consider:
+
+1. JS request / response
+2. SAS inputs / outputs
+3. Configuration
+
+### JS Request / Response
+
+To install the library you can simply run `npm install sasjs` or include a `<script>` tag with a reference to our [CDN](https://www.jsdelivr.com/package/npm/sasjs).
```

Comment:
>npm WARN deprecated sasjs@2.20.1: sasjs has been deprecated. Please use https://www.npmjs.com/package/@sasjs/adapter instead.

Reply:
`npm install @sasjs/adapter`

Reply:
JSDelivr link needs to be updated to reflect that as well.

Reply:
good spot!!